package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
-    "lint": "xo",
+    "lint": "eslint --ext .js,.json .",
```

Comment:
switching eslint to xo seems like a good but separate PR?

Reply:
I can do that.